solution/test/test-sum-evens.js

```diff
@@ -0,0 +1,26 @@
+const sumEvens = require('../src/sum-evens');
+const assert = require('assert');
+
+it("sum evens", () => {
+    const numbers = [1, 2, 3, 4, 5, 6];
+    assert.equal(sumEvens(numbers), 12);
+});
+
+it("sum evens when no evens", () => {
+    const numbers = [1, 3, 5, 1001];
+    assert.equal(sumEvens(numbers), 0);
+});
+
+it("sum evens when negative numbers", () => {
+    const numbers = [-10, -12, -11, -10, -9];
+    assert.equal(sumEvens(numbers), -32);
+});
+
+it("sum evens when non array", () => {
+    assert.equal(sumEvens("not an array"), undefined);
+});
+
+it("sum evens when non integer values", () => {
+    const nonIntegers = [10, "definitely not an integer", 2, 3, 3.3];
+    assert.equal(sumEvens(nonIntegers), undefined);
+});
```

Comment:
![image](https://user-images.githubusercontent.com/2707601/46124932-5f203d80-c22f-11e8-9d5c-96ae3661b566.png)

Reply:
also the problem states:

Given an **integer array** of size N, compute the sum of all even numbers in this array